src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenTuples.vb

```diff
@@ -6744,6 +6744,188 @@ End Module
         End Sub
 
         <Fact>
```

Comment:
Thanks! There is a lot of tests there. We should port them.
At this point, since this change is mostly a port from C#, we have some extra confidence that it works. Also it is blocking some EE work, so test completeness is not as blocking.

I have entered an issue for porting the AttributeTests_Tuples.cs tests to VB. [https://github.com/dotnet/roslyn/issues/13942](https://github.com/dotnet/roslyn/issues/13942)

---
In reply to: [79723061](https://github.com/dotnet/roslyn/pull/13923#discussion_r79723061) [](ancestors = 79723061)